test/browser_tests/spec_suites/homepage.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+function _getStylesheets() {
+  var sheets = document.styleSheets;
+  var hrefs = [];
+
+  for ( var i = 0; i < sheets.length; i++ ) {
+    hrefs.push( sheets[i].href );
+  }
+
+  return hrefs;
+}
+
+describe( 'The homepage', function() {
+
+  beforeEach( function() {
+    browser.get( '/' );
+  } );
+
+  it( 'should load the stylesheet', function() {
+    browser.executeScript( _getStylesheets ).then(
+      function( styleSheets ) {
+        browser.styleSheets = styleSheets;
+
+        browser.getCapabilities().then( function( cap ) {
+          browser.name = cap.get( 'browserName' );
```

Comment:
You mean the nesting of `then()` calls? Yeah I wasn't 100% certain either, but the stylesheets weren't coming through as I was expecting and with this change they're all good.
I get `Expected undefined to contain 'http://localhost:8000/static/css/main.css'` if I don't nest them, so I figured it was a problem with the expect() call before the promise resolved.